subtests/docker_cli/run/run_install.py

```diff
@@ -20,10 +20,13 @@ def postprocess(self):
         # ancestor method must have been successful
         self.sub_stuff['containers'].append(name)
 
-        mustpass(DockerCmd(self, 'commit', [name, name]).execute())
+        # images with same names as containers are confusing
+        eman = name[-1::-1]
```

Comment:
...ahh, you're right.  It's done for images but not for container names.  good catch.